packages/accounts-base/accounts_client.js

```diff
@@ -191,7 +191,7 @@ Ap.callLoginMethod = function (options) {
       });
     } else {
       self._onLoginFailureHook.each(function (callback) {
-        callback();
+        callback({ error: error });
```

Comment:
Can you maybe just change this to use shorthand notation (`callback({ error })`), and see if the CircleCI tests pass after rebasing?